src/main/java/com/pgasync/Connectible.java

```diff
@@ -0,0 +1,15 @@
+package com.pgasync;
```

Comment:
Why the new package structure?

Reply:
The new package structure is proposed to get rid of clumsy name `impl` and to be able to give an implementation a name like a `github`.